pgbedrock/attributes.py

```diff
@@ -60,7 +57,6 @@
 def analyze_attributes(spec, cursor, verbose):
     logger.debug('Starting analyze_attributes()')
     dbcontext = DatabaseContext(cursor, verbose)
-    fail_if_undocumented_roles(spec, dbcontext)
```

Comment:
what's the reason for removing this failure case?

Reply:
It's moved into the spec_inspector.py, for a few reasons:
1. That way all spec validation code is in one module together
2. If there's an issue with the spec it'd be nice to fail immediately vs. go through a bunch of submodules and _then_ fail, which is what happens currently. By co-locating this all in the spec_inspector.py, when we load the spec we check the quality of the spec and can give that immediate feedback.